src/database/parse-server.ts

```diff
@@ -23,6 +25,16 @@ export default class ParseServer implements Database {
     return this.instance;
   }
 
+  public async createEvent(eventModel: EventModel): Promise<any> {
+    const event = new this.Event();
+    return event.save(eventModel);
+  }
+
+  public async getEvent(objectId: string): Promise<any> {
+    const query = new this.Parse.Query(this.Event);
+    return query.get(objectId);
+  }
+
   public async getEvents(
     numResults: number,
     pageNumber: number,
```

Comment:
when will we need to get by numResults / pageNumber?

Reply:
I assume we're not trying to load all the results at once. This allows us to have either actual pages, or an infinite scroll (when the user reaches the end of the list, load the next "page", but in the same list view).

Reply:
oh im use that that being called contextToken and maxResults